### PR TITLE
Add United Nations World Data Forum webinar

### PR DESCRIPTION
**Pull request by 454de6e**
Closes GH-216.

### CODE REVIEW

**Comment by 454de6e** [comment]
https://covid-19-response-git-gh-216.unstats.now.sh/sharing/data-in-the-fight-against-covid-19/

File: content/sharing/data-in-the-fight-against-covid-19.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+end: 2020-04-29T10:00
+timezone: America/New_York
+registrationLink: https://docs.google.com/forms/d/e/1FAIpQLScr8avLnEzrL5wEf2eLWyuBZT7p_cYwy8lu7IJe8Dj_uHAY-w/viewform
+---
+
+In the midst of the current global COVID-19 crisis, there is an unprecedented
+need for data to not only enable decision makers to inform policies and planning
+but to also minimize the risk for all especially the vulnerable population
+groups. For this purpose, timely data disaggregated by age, sex, location and
+other markers is important to inform decision makers and the general public.
```
Comment (gonzalezmorales) [inline content/sharing/data-in-the-fight-against-covid-19.md:13]: Add this and that

File: content/sharing/data-in-the-fight-against-covid-19.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (gonzalezmorales) [inline content/sharing/data-in-the-fight-against-covid-19.md:14]: bla bla

File: content/sharing/data-in-the-fight-against-covid-19.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+From the point of view of the statistical community, are we ready for such a
+huge challenge? What do we learn from this challenge and experiences?
+Governments are working together with partners at the national and global level
+to deal with the current emergency by adapting innovative production methods and
```
Comment (gonzalezmorales) [inline content/sharing/data-in-the-fight-against-covid-19.md:18]: 2nd comment

**Comment by 454de6e** [comment]
Closing to create a new one for demo purposes.

**Pull request closed without merging**